src/frontend/js/classes/sidebar-filter.js

```diff
@@ -0,0 +1,341 @@
+/**
+ * HEIGHT of the header in px
+ */
+const HEADER_HEIGHT = 56;
```

Comment:
you can access CSS variable instead of hardcoding the value
Try this:
```js
const headerHeidht = parseInt(window.getComputedStyle(document.documentElement)
    .getPropertyValue('--layout-height-header'));
```

Reply:
done